rust-toolchain.toml

```diff
@@ -1,4 +1,4 @@
 [toolchain]
-channel = "nightly-2023-01-13" 
+channel = "nightly-2023-05-22"
```

Comment:
I am actually surprised that it works. Earlier srtool images returned an error because it only has the stable toolchain installed, which conflicted with the one here. Don't know what changed on their end.